solutions/coreclr/run-single-test.sh

```diff
@@ -1,43 +1,26 @@
 #!/bin/bash
 
-USAGE="$0 path-to-myst null|exec-sgx|exec-linux config.json timeout ext2|cpio|package <path-to-test-dll>"
+USAGE="$0 path-to-myst null|exec-sgx|exec-linux config.json timeout <path-to-test-dll> <optional-extra-params>"
 ret_val=-1
 t0=$(date +"%s")
 TIMEOUT=$4
 
-if [[ "$5" == "package" ]]; then
```

Comment:
The idea behind packaged runs was that it would be faster. Is there any difference in the time taken to run the suite?